src/modules/ce/components/unit/PrioritizedClientsTable.tsx

```diff
@@ -81,18 +86,49 @@ const PrioritizedClientsTable: React.FC<Props> = ({ opportunity }) => {
     ];
   }, [project.access.canStartReferrals, opportunity, status]);
 
+  // If CandidatePool has not been generated yet (due to change in eligibility or prioritization requirements), show a message
+  if (!opportunity.candidatesGeneratedAt) {
+    return (
+      <Alert severity='info'>
+        The waitlist for this unit has not been generated yet. Please check back
+        later.
+      </Alert>
+    );
+  }
+  const candidatesGeneratedAt = parseHmisDateString(
```

Comment:
Could use `RelativeDateDisplay`'s `getFormattedDates` helper? Maybe move that out into the `hmisUtil`? Seems like getting the relative and absolute date from the hmis date string is going to become an increasingly common pattern in the code (since our design library recommends it almost everywhere dates are shown), so I like the idea of having a common helper that does both

Reply:
Yes that sounds like a good future improvement, going to merge as is for now